.github/workflows/release-docs.yml

```diff
@@ -0,0 +1,61 @@
+---
+name: "Release Documentation"
+
+"on":
+  push:
+    branches:
+      - 'develop'
+      - '#5670'
+
+jobs:
+  gen-docs:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Check out vega code
+        uses: actions/checkout@v3
+        with:
+          path: 'vega'
+      -
+        name: "Setup ssh key for PR"
+        uses: webfactory/ssh-agent@v0.5.4
+        with:
+          ssh-private-key: ${{ secrets.DOCUMENTATION_KEY }}
+      -
+        name: Check out docs code
+        uses: actions/checkout@v3
+        with:
+          path: 'documentation'
+          repository: 'vegaprotocol/documentation'
+          ssh-key: ${{ secrets.DOCUMENTATION_KEY }}
+      -
+        name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.18
+      -
+        name: Get dependencies
+        run: cd vega; bash script/gettools.sh
+      -
+        name: Generate docs
+        run: cd vega; make proto_docs; echo "generated docs:"; find protos/generated -type f
+      -
+        name: Make PR to the docs repo
+        env:
+          GH_TOKEN: ${{ secrets.VEGA_CI_BOT_GITHUB_TOKEN }}
+        run: |
+          mkdir -p documentation/specs/$GITHUB_REF_NAME
```

Comment:
In https://github.com/vegaprotocol/documentation/pull/216, the folder name comes out as '#issuenumber' not 'v0.54.0'

Reply:
GITHUB_REF_NAME is branch name of tag name depending on the event, in this PR branch name is issue number, that's why it confuses. I will just parse `git tag` for that instead